tests/src/JIT/HardwareIntrinsics/X86/Shared/SimpleBinOpTest.template

```diff
@@ -0,0 +1,209 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
The all generated tests (and therefore the template) should contain a comment noting that the test is generated and should not be manually edited.  It should also contain instructions on how to regenerate.